generation/framework/htcondor/HTCondor.py

```diff
@@ -11,16 +13,22 @@
 
 
 class HTCondorWorkflow(law.htcondor.HTCondorWorkflow):
+    ConfigParser = luigi.configuration.cfg_parser.LuigiConfigParser.instance()
+    # ConfigParser = law.config.Config.instance()
```

Comment:
Delete this commented out line?

Reply:
I would leave it since the commented out parts are the equivalent way of access to the configuration via the law config. Newer developments in law merge the `luigi.cfg` and `law.cfg` into a single file. So the commented out line will eventually replace the line above, where the luigi configuration is accessed for now.